pages/docs/form/editable.mdx

```diff
@@ -90,6 +90,54 @@ function CustomControlsExample() {
 }
 ```
 
+### With custom input and controls
+
+Building on the last example, you may want to customise the input as well. I've also tweaked the UI, by adding a hover color, and a tooltip. As you can see, we can change the input by setting the `as` prop. This design is inspired by the [Atlassian inline edit](https://atlassian.design/components/inline-edit/examples) component.
```

Comment:
I would rephrase this like this: 
You may want to customize the `EditableInput` as well. This can be achieved by using a custom `Input` component in the `as` prop.
// example